src/builder.ts

```diff
@@ -170,7 +170,13 @@ export const UNKNOWN_TYPE_SCALAR = decorateType(
   }
 );
 
+export interface Plugin {
+  // TODO TS 3.6 Generator type
+  onBuild?: (types: any, builder: SchemaBuilder) => Generator; //Generator<void, void, void>;
```

Comment:
Did we need a generator for some reason?

Reply:
Nope could not. If we discover need for `onAfterBuild` we can revisit in future I guess.